pkgs/servers/sql/postgresql/ext/tds_fdw.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, postgresql, freetds }:
+
+stdenv.mkDerivation rec {
+  name = "tds_fdw-${version}";
```

name will automatically be generated if you do:
```suggestion
  pname = "tds_fdw";
```

---

Done!